pubspec.yaml

```diff
@@ -15,8 +15,10 @@ dependencies:
   instabug_flutter: '>=11.0.0 <12.0.0'
 
 dev_dependencies:
+  build_runner: ^2.0.3
   flutter_test:
     sdk: flutter
+  mockito: 5.2.0
```

Comment:
**suggestion (non-blocking):** We can try using [`mocktail`](https://pub.dev/packages/mocktail), instead of `mockito`, this will enable us to mock `InstabugHostApi` without much changes as `mockito`.

Reply:
I think it'd be better if we stick to one library when it comes to mocking, to have a consistent way between our different Flutter packages and avoid any conflicts that may arise.